servicetalk-encoding-netty/src/main/java/io/servicetalk/encoding/netty/DefaultContentCodecBuilder.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright © 2021 Apple Inc. and the ServiceTalk project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.servicetalk.encoding.netty;
+
+import io.servicetalk.encoding.api.ContentCodec;
+
+import io.netty.handler.codec.compression.JdkZlibDecoder;
+import io.netty.handler.codec.compression.JdkZlibEncoder;
+import io.netty.handler.codec.compression.ZlibWrapper;
+
+import static io.servicetalk.buffer.api.CharSequences.newAsciiString;
+
+abstract class DefaultContentCodecBuilder implements ContentCodecBuilder {
```

Comment:
After removing snappy, there is no value in this `DefaultContentCodecBuilder` top level class. Consider moving `ZipContentCodecBuilder` to the top level